### ISSUE TITLE
Verify copyright of music

### ISSUE THREAD

**Issue by monacofj** [courselab/coral#145]
Directory `coral/musics` has three audio files:

- `CPU Talk - FMA - CC BY BoxCat Games.mp3`
- `game_over.wav`  
- `got_apple.ogg`

Make sure they are FOSS and provide full copyright and licensing information (make sure it's FOSS) according to SPDX [1] specifications. Preferably, use the `REUSE` [2] tool for the task.

Under  `coral/musics`, create the text files

- `CPU Talk - FMA - CC BY BoxCat Games.mp3.license`
- `game_over.wav.license`  
- `got_apple.ogg.license`

The content of the license file should be like

```
   SPDX-FileCopyrightText: <YEAR COPYRIGHT NAME>
  
   SPDX-License-Identifier: <LICENSE>
```

[1] https://spdx.dev/
[2] https://reuse.software/

**Comment by joaoduarteN** [comment]
i can take this one!

**Comment by joaoduarteN** [comment]
I can do this one! Closed by mistake, sorry

**Comment by monacofj** [comment]
It seems that @MarcusV-Silva added the files.

**Comment by de-abreu** [comment]
Found the "game over" sound, it is licensed under C0: https://freesound.org/people/EVRetro/sounds/533034/
Here's another one that is free for commercial use and could be equivalent to the one we're currently using: https://www.zapsplat.com/music/retro-8-bit-video-game-object-collect-or-pick-up/

**Comment by monacofj** [comment]
> Here's another one that is free for commercial use and could be equivalent to the one we're currently using when an a fruit is eaten: https://www.zapsplat.com/music/retro-8-bit-video-game-object-collect-or-pick-up/

It seems that this one is **not open source**. As far as I could trace, the license is 

https://www.zapsplat.com/license-type/standard-license/

**Comment by de-abreu** [comment]
Yes, that is the license. I've read through it and, unless I am mistaken, it is equivalent to a CC-BY.

**Comment by monacofj** [comment]
Am I missing something, or this is the license?
It does not looks like FOSS.
```
What you can not use our free sound or music effects for:
You may not use our sound effects for the following purposes under any circumstance:

Our sound effects and music must not be shared with any person.
Our sound effects and music must not be distributed in any form including on other websites, on social networks, file sharing platforms, CD, DVD or any other ROM.
Our sound effects and music can not be loaned, rented, sub-licensed or sold to any third party.
Our sound effects can not be edited to form new sound effects for sale or distribution outside of your project.
Please do not share or share links to download our sounds and music to YouTube or any other video or audio network where the primary value is to share our sounds or music outside of a production.
Relaxation videos where our sounds and music are the primary value of the work.
You can not remix our music including adding new instrumentation to form a new music composition.
You may not use our sound effects or music files in a physical product such as a toy without a seperate Agreement. Please contact us.
Using our sound effects or music tracks to train Artificial Intelligence (AI) or any deep learning model is prohibited.
```

**Comment by de-abreu** [comment]
No, you're right. Even though it can be modified and used commercially, there are all these restraints placed upon sharing the original file. Which would invariably happen if we placed it over here.

**Comment by de-abreu** [comment]
Here's some others:
https://freesound.org/people/n_audioman/sounds/275897/ (CC0)
https://freesound.org/people/ryusa/sounds/531176/ (CC-BY)
https://freesound.org/people/OwlStorm/sounds/404774/ (CC0)

Wow, that wesite is quite good, you can filter results by licenses!

**Comment by de-abreu** [comment]
> The music "CPU talk" I downloaded from this link https://freemusicarchive.org/music/BoxCat_Games/Nameless_the_Hackers_RPG_Soundtrack , and the other two sound effects I took from these links https://freesound.org/people/plasterbrain/sounds/242857/ and https://freesound.org/people/EVRetro/sounds/533034/

Added the licensing information to the audios that you've found, Marcus. Issue solved by PR #200 .
